.env.example

```diff
@@ -18,42 +27,33 @@ NEXT_PUBLIC_API_HOST=http://localhost:3000
 NEXT_PUBLIC_EVENTS_API=http://localhost:3334
 NEXT_PUBLIC_HOME_URL=http://localhost:3000
 
-# Don't forget to deploy new contracts by using /parity route
+# Don't forget to deploy new contracts by using /administration route
```

Comment:
I think that this page don't have the deployment of contracts anymore, maybe we can remove this comment, as now contracts deployes are done by the deploy-v2 script